src/components/CryptoItem/CryptoItem.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {
+  ItemContainer,
+  ItemLeftContainer,
+  ItemRigthContainer,
+  ItemSubSubContainer,
+  ItemTitle,
+  StyledArrow,
+  StyledImage,
+  TextPercentage,
+} from './style';
+import ArrowUp from '../../assets/ArrowGreen.png';
+import ArrowDown from '../../assets/ArrowRed.png';
+import {Text} from 'react-native';
+import DataTypes from '../../types/types';
+
+const CryptoItem = ({
+  symbol,
+  name,
+  value,
+  percentage,
+  signal,
+  slug,
+  image,
+}: {
```

Comment:
create a type for this object

Reply:
actualy, the type should be DataTypes, change the name for a more descriptive one, like CryptoTypes